src/client/extension.ts

```diff
@@ -192,9 +194,14 @@ export async function activate(
             const orgID = orgDetails.OrgId;
             const artemisResponse = await ArtemisService.fetchArtemisResponse(orgID, _telemetry);
             if (artemisResponse !== null && artemisResponse.length > 0) {
+                const pacActiveAuth = await pacTerminal.getWrapper()?.activeAuth();
+                let AadIdObject;
+                if ((pacActiveAuth && pacActiveAuth.Status === SUCCESS)) {
+                    AadIdObject = pacActiveAuth.Results.filter(obj => obj.Key === AadIdKey);
+                }
                 const { geoName, geoLongName } = artemisResponse[0]?.response as unknown as IArtemisAPIOrgResponse;
                 oneDSLoggerWrapper.instantiate(geoName, geoLongName);
-                oneDSLoggerWrapper.getLogger().traceInfo(desktopTelemetryEventNames.DESKTOP_EXTENSION_INIT_CONTEXT, { ...orgDetails, orgGeo: geoName });
+                oneDSLoggerWrapper.getLogger().traceInfo(desktopTelemetryEventNames.DESKTOP_EXTENSION_INIT_CONTEXT, { ...orgDetails, orgGeo: geoName, AadId: AadIdObject?.[0]?.Value });
```

Comment:
do you want to include aadId in cases where value can be null?

Reply:
Updated